shared/CMakeLists.txt

```diff
@@ -25,17 +25,16 @@ set (srcs
 
     String.F90
   )
-esma_add_library (${this} SRCS ${srcs} DEPENDENCIES gftl-shared pflogger MPI::MPI_Fortran)
-#target_include_directories (${this} PRIVATE ${MAPL_SOURCE_DIR}/include)
+esma_add_library (${this} SRCS ${srcs} DEPENDENCIES gftl-shared MPI::MPI_Fortran pflogger)
 target_include_directories (${this} PUBLIC
           $<BUILD_INTERFACE:${MAPL_SOURCE_DIR}/include>)
 
 #file (COPY MAPL_ErrLogMain.h DESTINATION ${esma_include}/${this})
 #file (COPY MAPL_ErrLog.h DESTINATION ${esma_include}/${this})
 #file (COPY MAPL_Exceptions.h DESTINATION ${esma_include}/${this})
 
-#if (PFUNIT_FOUND)
-  #add_subdirectory (tests)
-#endif ()
+if (PFUNIT_FOUND)
+  add_subdirectory (tests)
+endif ()
```

Comment:
I think these may have been commented out for a reason.   We will test and see.

Reply:
Yeah, I think these lines were done that way for a reason. But I suppose we could...combine them a la:
```cmake
if (PFUNIT_FOUND)
  add_subdirectory(tests EXCLUDE_FROM_ALL)
endif ()
```
?

Reply:
This is what is done in `profiler/CMakeLists.txt` it looks like.

Reply:
Hmm. Now that I look around, we might have other places in MAPL to do this. `pfio/CMakeLists.txt` seems a candidate.